std/algorithm/searching.d

```diff
@@ -1204,6 +1210,75 @@ bool endsWith(alias pred, R)(R doesThisEnd)
     }
 }
 
+/**
```

Comment:
Why? It's set to `private`, so it will never be exposed to the user and for Phobos devs looking at the source it's nice to have some documentation.